src/ui/atoms/button/button.scss

```diff
@@ -12,19 +12,15 @@ $background-colors: (
 .okp4-button-main {
   @include with-theme {
     cursor: pointer;
-    text-decoration: unset;
     border: unset;
     min-height: 40px;
-    padding: 10px 20px;
-    display: flex;
-    justify-content: space-around;
-    align-items: center;
+
     min-width: 65px;
     max-width: 273px;
     overflow: hidden;
 
     &.variant-secondary {
-      border-radius: 50px;
+      border-radius: 50%;
```

Comment:
By doing this, the `secondary` variant is always rounded... 
![image](https://user-images.githubusercontent.com/47554752/181246652-6f809bf5-7f58-4e29-a8a5-37784b2b09e7.png)

```suggestion
      border-radius: 24px;
```

Reply:
Unfortunate miss by me, will fix 👍